src/app/views/marketplace/marketplace-app.controller.coffee

```diff
@@ -41,6 +41,14 @@ angular.module 'mnoEnterpriseAngular'
 
         vm.isLoading = false
 
+      # Update the comments and avergae review
+      vm.updateReview = (response) ->
+        # Select and push to comments the last comment added
+        [first, ..., newReview] = response.app.reviews
```

Comment:
As mentioned in the previous comment, you don't need `first`, so you can decompose the array this way:
`[..., newReview] = response.app.reviews`